src/main/java/com/_up/megastore/services/implementations/SizeService.java

```diff
@@ -41,4 +41,19 @@ public SizeResponse updateSize(UUID sizeId,UpdateSizeRequest updateSizeRequest){
 
         return SizeMapper.toSizeResponse(sizeRepository.save(size));
     }
+
+    @Override
+    public void deleteSize(UUID sizeId){
+        Size size = findSizeByIdOrThrowException(sizeId);
+        ifSizeIsNotDeletedThrowException(size);
+        size.setDeleted(true);
+
+        sizeRepository.save(size);
+    }
+
+    public void ifSizeIsNotDeletedThrowException(Size size){
```

Comment:
El nombre del método está mal porque se lanza la excepción SI ESTÁ eliminado

```suggestion
    public void ifSizeIsDeletedThrowException(Size size){
```